msteams-platform/index.yml

```diff
@@ -139,7 +166,7 @@ tools:
       imageSrc: /media/logos/logo_vs-ide.svg
```

Comment:
Icons are not displaying for the VS Code and VS cards.

Reply:
They would get displayed, once the hub-page is live.